changelog/changelog.ddoc

```diff
@@ -32,14 +32,14 @@ BUGSTITLE = <div class="bugsfixed">$(H4 $1) $(OL $2 )</div>
 UPCOMING = <div id="upcoming">$(H4 Under Construction) $(OL $0 )</div>
 WHATSNEW = <div id="whatsnew">$(H4 New/Changed Features) $(UL $0 )</div>
 NEWPHOBOS = <div id="whatsnew">$(H4 New/Changed Phobos) $(UL $0 )</div>
-LARGE=<font size=4>$0</font>
+LARGE=<font size="4">$0</font>
 TRENAME=$(TR $(TD $(CODE $1)) $(TD $(CODE $2)))
 LII= $(LI $(D $1) $(UL $+ ))
 
-WHATSNEW2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2&query_format=advanced&chfield=resolution&chfieldfrom=$1&chfieldvalue=FIXED&bug_severity=enhancement&bug_status=RESOLVED&version=D2&version=D1%20%26%20D2&resolution=FIXED&product=D, New/Changed Features)
-DMDBUGSFIXED2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2&query_format=advanced&chfield=resolution&chfieldfrom=$1&chfieldvalue=FIXED&bug_severity=regression&bug_severity=blocker&bug_severity=critical&bug_severity=major&bug_severity=normal&bug_severity=minor&bug_severity=trivial&bug_status=RESOLVED&version=D2&version=D1%20%26%20D2&component=DMD&resolution=FIXED&product=D, DMD Bugs Fixed)
-DRUNTIMEBUGSFIXED2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2&query_format=advanced&chfield=resolution&chfieldfrom=$1&chfieldvalue=FIXED&bug_severity=regression&bug_severity=blocker&bug_severity=critical&bug_severity=major&bug_severity=normal&bug_severity=minor&bug_severity=trivial&bug_status=RESOLVED&version=D2&version=D1%20%26%20D2&component=druntime&resolution=FIXED&product=D, Druntime Bugs Fixed)
-PHOBOSBUGSFIXED2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2&query_format=advanced&chfield=resolution&chfieldfrom=$1&chfieldvalue=FIXED&bug_severity=regression&bug_severity=blocker&bug_severity=critical&bug_severity=major&bug_severity=normal&bug_severity=minor&bug_severity=trivial&bug_status=RESOLVED&version=D2&version=D1%20%26%20D2&component=Phobos&resolution=FIXED&product=D, Phobos Bugs Fixed)
+WHATSNEW2=$(LINK2 https://issues.dlang.org/buglist.cgi?chfieldto=$2$(AMP)query_format=advanced$(AMP)chfield=resolution$(AMP)chfieldfrom=$1$(AMP)chfieldvalue=FIXED$(AMP)bug_severity=enhancement$(AMP)bug_status=RESOLVED$(AMP)version=D2$(AMP)version=D1%20%26%20D2$(AMP)resolution=FIXED$(AMP)product=D, New/Changed Features)
```

Comment:
Yes, all ampersands in HTML must be escaped unless they start an entity. `<script>` tags get a pass as a special exception, I think.